public/v4/apps/formio.yml

```diff
@@ -0,0 +1,94 @@
+        volumes:
+            - $$cap_appname-db-data:/data/db
+        restart: always
+        environment:
+            MONGO_INITDB_ROOT_USERNAME: $$cap_db_user
+            MONGO_INITDB_ROOT_PASSWORD: $$cap_db_pass
+        caproverExtra:
+            notExposeAsWebApp: 'true'
+    $$cap_appname:
+        documentation: Taken from https://hub.docker.com/r/calipseo/formio
+        depends_on:
+            - $$cap_appname-mongodb
+        image: calipseo/formio:$$cap_formio_version
+        volumes:
+            - $$cap_appname-data:/app
+        restart: always
+        environment:
+            APP_URL: 'https://$$cap_appname.$$cap_root_domain'
+            API_URL: 'https://$$cap_appname.$$cap_root_domain'
+            MONGO: mongodb://$$cap_db_user:$$cap_db_pass@srv-captain--$$cap_appname-mongodb:27017/formio
+            ROOT_EMAIL: $$cap_admin_email
+            ROOT_PASSWORD: $$cap_admin_pass
+            SMTP_HOST: $$cap_smtp_host
+            SMTP_PORT: $$cap_smtp_port
+            SMTP_USERNAME: $$cap_smtp_user
+            SMTP_PASSWORD: $$cap_smtp_pass
+        caproverExtra:
+            containerHttpPort: '3001'
+caproverOneClickApp:
+    variables:
+        - id: $$cap_admin_email
+          label: Formio default admin email
+          defaultValue: admin@example.com
+          description: Admin email for Formio.
+          validRegex: /.{1,}/
+        - id: $$cap_admin_pass
+          label: admin password
+          description: Password for the admin user.
+          validRegex: /.{1,}/
+        - id: $$cap_formio_version
+          label: Formio Version
+          defaultValue: 1.90.7
+          description: https://hub.docker.com/r/calipseo/formio/tags
+          validRegex: /^([^\s^\/])+$/
+        - id: $$cap_mongo_version
+          label: Mongo (database) version
+          defaultValue: 4.1
+          description: Check out their Docker page for the valid tags https://hub.docker.com/_/mongo?tab=tags
+          validRegex: /^([^\s^\/])+$/
+        - id: $$cap_db_pass
+          label: database password
+          description: Password for the database user and root using mongodb.
+          validRegex: /.{1,}/
+        - id: $$cap_db_name
+          label: database name
+          defaultValue: formio
+          description: name for the database.
+          validRegex: /^([a-zA-Z0-9])+$/
+        - id: $$cap_db_user
+          label: database user
+          defaultValue: formio
+          description: Username for the database using mongodb.
+          validRegex: /^([a-zA-Z0-9])+$/
+        - id: $$cap_smtp_host
+          label: SMPT host
+          defaultValue: smtp.gmail.com
+          description: The host for your smtp server
+        - id: $$cap_smtp_port
+          label: SMTP port
+          defaultValue: '587'
+          description: The port for your smtp server
+        - id: $$cap_smtp_user
+          label: SMTP user
+          defaultValue: your_email@gmail.com
```

Comment:
Remove the default value to indicate that the user needs to modify this, maybe move it to label.

Same with other default values that are not actually default values, and they are just examples. Let's move them to "label"

Reply:
Thanks! I decided to remove all the default values for the SMTP as they already have a label